src/bin/vip-import-sql.js

```diff
@@ -204,15 +207,31 @@ command( {
 		console.log( `  importing: ${ chalk.blueBright( fileName ) }` );
 		console.log( `         to: ${ chalk.cyan( domain ) }` );
 		console.log( `       site: ${ app.name } (${ formatEnvironment( opts.env.type ) })` );
-		searchReplace ? '' : console.log();
+		if ( searchReplace && searchReplace.length ) {
+			const searchAndReplaceParams = searchReplace.split( ',' );
+			console.log(
+				`        s-r: ${ chalk.blue( searchAndReplaceParams[ 0 ] ) } -> ${ chalk.blue(
+					searchAndReplaceParams[ 1 ]
+				) }`
+			);
+		}
+		console.log();
+
+		// NO `console.log` after this point! It will break the progress printing.
 
 		let fileNameToUpload = fileName;
 
+		const progressTracker = new ProgressTracker( SQL_IMPORT_PREFLIGHT_PROGRESS_STEPS );
+		progressTracker.prefix = `=============================================================
+Processing your sql import for env ID: ${ env.id }, app ID: ${ env.appId }:\n`;
+		progressTracker.suffix = `\n${ getGlyphForStatus(
+			'running',
+			progressTracker.runningSprite
+		) } Loading remaining steps`;
```

Comment:
Noting that I missed spotting this log the first time around, so it may not be too noticeable (although that may be because we didn't bring in the fast spinner changes yet).